src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.test.tsx

```diff
@@ -65,7 +65,33 @@ describe('IncidentsDetail', () => {
       )
     )
 
-    expect(screen.getByText("Foto's")).toBeInTheDocument()
+    expect(screen.getByText(/Foto's/)).toBeInTheDocument()
+
+    const attachmentMunicipality = incidentsDetail._links[
+      'sia:attachments'
+    ].map((attachment) => ({
+      ...attachment,
+      created_by: 'Gemeente Amsterdam',
```

Comment:
If the creator is an officer of the municipality, then the created_by should be empty. If you don't mean the officer, then maybe pick a different name? It's kinda confusing now (even though it's just a test)

Reply:
changed